src/ucp/core/ucp_ep.h

```diff
@@ -438,6 +438,10 @@ struct ucp_ep_config {
 };
 
 
+/* Forward declaration */
+typedef struct ucp_ep_ext ucp_ep_ext_t;
```

Comment:
can we define ucp_ep_ext struct before ucp_ep to avoid this?

Reply:
Then we'll have to forward-declare `ucp_ep`, because the `ext` has a pointer to it.

Reply:
https://github.com/openucx/ucx/blob/3d4ccf50de87c2e1652da440e3fb27b6cfa71490/src/ucp/api/ucp_def.h#L92